web/src/message_list.js

```diff
@@ -405,6 +405,8 @@ export class MessageList {
         $recipient_row.find(".stream_topic").hide();
         $recipient_row.find(".topic_edit").show();
         $recipient_row.find(".always_visible_topic_edit").hide();
+        $recipient_row.find(".on_hover_topic_resolve").hide();
+        $recipient_row.find(".zulip-icon-mute").hide();
```

Comment:
Also, we should hide mark as `unresolve`.

Reply:
Done.